Week3/homework/step2-2.js

```diff
@@ -2,19 +2,23 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+  console.log(numbers);
+  for (const number of numbers) {
+    if (number % 3 === 0) threeCallback(number);
```

Comment:
I would recommend that you always use curly braces around the `if` block, even it contains one statement only. This makes it easy to add more statements later. But if you nevertheless leave the curly braces out then I prefer that you do it the way you have done. The example below is particularly dangerous:

```js
if (number % 3 === 0)
  threeCallback(number);
```
This is because you might be tempted to add something, believing that is part of the `if` block, as suggested by the indentation:

 ```js
if (number % 3 === 0)
  console.log(number);
  threeCallback(number);
```